Selenium.Axe/Selenium.Axe/Selenium.Axe.csproj

```diff
@@ -10,10 +10,10 @@
     <PackageTags>selenium axe ada webdriver</PackageTags>
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
     <RepositoryType>git</RepositoryType>
-    <AssemblyVersion>0.0.0.1</AssemblyVersion>
+    <VersionPrefix>1.1.0.0</VersionPrefix>
```

Comment:
This actually only increased minor, but it wasn't very clear because it moved the point where the "true" release version was applied from azure-pipeline.yml (which originally overrode this as 1.0.1) to this file.

This being unclear is a good point; I've updated it now to switch back to having azure-pipelines.yml apply the true version, and left a comment here explaining it